scripts/index.js

```diff
@@ -0,0 +1,81 @@
+const upClick = document.getElementsByClassName('upClick')[0];
+const temperature = document.getElementsByClassName('temperature')[0];
+const downClick = document.getElementsByClassName('downClick')[0];
+let parseTemperature= parseInt(temperature.innerHTML)
+
+const handleColorChange=()=>{
```

Comment:
You are doing more than handling the color change here. I would suggest a different name for this function.